src/payments/entities/payment-intent.entity.ts

```
@@ -0,0 +1,87 @@
+
+  @Column({ type: 'uuid', name: 'user_id' })
+  userId: string;
+
+  @Column({
+    type: 'enum',
+    enum: PaymentPurpose,
+    name: 'purpose',
+  })
+  purpose: PaymentPurpose;
+
+  @Column({ type: 'decimal', precision: 10, scale: 2 })
```

ESLint

[error] Replace `'decimal'` with `"decimal"`

(prettier/prettier)